src/main/java/com/hhplus/precourse/post/controller/UpdatePostController.java

```diff
@@ -0,0 +1,74 @@
+package com.hhplus.precourse.post.controller;
+
+import com.hhplus.precourse.common.web.ApiResponse;
+import com.hhplus.precourse.post.service.UpdatePostService;
+import com.hhplus.precourse.post.vo.PostVo;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotBlank;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDateTime;
+
+@RestController
+@RequiredArgsConstructor
+public class UpdatePostController {
+    private final UpdatePostService service;
+
+    @PutMapping("/posts/{id}")
+    ApiResponse<Response> update(@PathVariable Long id,
+                                 @Valid @RequestBody Request request) {
+        var command = convertToCommand(id, request);
+        var postVo = service.update(command);
+
+        return ApiResponse.success(
+            Response.from(postVo)
+        );
+    }
```

Comment:
```suggestion
    @PutMapping("/posts/{id}")
    ApiResponse<Response> update(@PathVariable Long id,
                                 @Valid @RequestBody Request request) {
        var command = request.toCommand(id);
        var postVo = service.update(command);

        return ApiResponse.success(
            Response.from(postVo)
        );
    }
```
create와 동일하게 도메인 내의 메서드를 호출하여 Command를 만들어줘도 좋을 것 같네요 ㅎ

Reply:
저 나름대로 to~~ 메서드는 외부 인자를 안받고 객체내 필드들로만 해결이 될때 만드는데요.
추천해주신거 보니까 아래쪽이 훨씬 깔끔하고 다른 클래스들이랑 통일감도 있고 좋네요ㅎㅎ
리뷰 반영해두겠습니다!